frontend/pages/club/[club]/renew.tsx

```diff
@@ -98,40 +98,54 @@ const PolicyBox = ({ onChecked = () => undefined }: Props): ReactElement => {
 
   const policies = [
     {
-      name: 'Campus Membership',
+      name: 'Nondiscrimination',
       content: (
         <div>
-          Membership in registered campus organizations must be open to all
-          persons without regard to race, color, sex, sexual or affectional
-          orientation, religion, national or ethnic origin, handicap, or
-          disability. Under Title IX of the U.S. Education Act Amendment of
-          1972, certain exemptions may be granted for intercollegiate and
-          intramural athletics, fraternities and sororities, and musical groups
-          based on vocal range. Members of all campus organizations must conduct
-          themselves at all times in a mature and responsible manner.
+          In alignment with the University of Pennsylvania’s Nondiscrimination
+          Statement, student-run organizations are expected to uphold a
+          commitment to non-discrimination. This means that these organizations
+          will not engage in discriminatory practices based on factors such as
+          race, color, sex, sexual orientation, gender identity, religion,
+          creed, national or ethnic origin, citizenship status, age, disability,
+          veteran status, or any other legally protected class status. It is
+          essential for student organizations to recognize the University’s
+          dedication to providing academic, social, and recreational programs
+          and services that are equally accessible to all, and as such, all
+          student-run organizations are required to conduct themselves and their
+          activities in accordance with this commitment.
         </div>
       ),
     },
     {
-      name: 'Legal Regulations',
+      name: 'Antihazing',
       content: (
         <div>
-          The rights and property of all persons are to be respected regardless
-          of time or place. Failure to comply with University, City, State, or
-          Federal laws and regulations can result in appropriate disciplinary
-          action. Members of campus organizations are expected to adhere to
-          standards of conduct established by Divisions and Departments of the
-          University.
+          Student organizations must fully comply with the University of
+          Pennsylvania’s Antihazing Regulations, as outlined in the{' '}
+          <a href="https://catalog.upenn.edu/pennbook/">Pennbook</a>. To ensure
```

Comment:
@julianweng lmk if we should put these links in branding

Reply:
I'm fine either way, probably not required since we don't have the Hub@Penn / Clubs distinction anymore and it's a one-time use.